apps/web/modules/ee/sso/lib/sso-handlers.ts

```diff
@@ -132,13 +137,24 @@ export const handleSSOCallback = async ({ user, account }: { user: TUser; accoun
         });
         isNewOrganization = true;
       }
-      const role = isNewOrganization ? "owner" : DEFAULT_ORGANIZATION_ROLE || "manager";
+      const role = isNewOrganization ? "owner" : DEFAULT_ORGANIZATION_ROLE || "owner";
       await createMembership(organization.id, userProfile.id, { role: role, accepted: true });
       await createAccount({
         ...account,
         userId: userProfile.id,
       });
 
+      if (DEFAULT_TEAM_ID) {
+        await createTeamMembership(
```

Comment:
pls check if this should be a call to `createDefaultTeamMembership` and change it if it needs to be 🙏